app/views/container_group/_main.html.haml

```diff
@@ -1,7 +1,10 @@
 = render :partial => "layouts/flash_msg"
 .row
   .col-md-12.col-lg-6
-    = render :partial => "shared/summary/textual", :locals => { :title => _("Properties"), :items => textual_group_properties }
+    = render :partial => "shared/summary/textual", :locals => {:title => _("Properties"),
+    :items => textual_group_properties}
```

Comment:
The only reason I broke the line was Rubocop "LineLength: Line is too long" alert.
I don't mind rolling back, what do you think of this way?

```Ruby
.col-md-12.col-lg-6
  = render :partial => "shared/summary/textual",
           :locals => {:title => _("Properties"), :items => textual_group_properties}
```